quda_interface.c

```diff
@@ -982,6 +1048,7 @@ int invert_doublet_eo_quda(spinor * const Even_new_s, spinor * const Odd_new_s,
     inv_param.solve_type = QUDA_NORMERR_SOLVE;
     if(g_proc_id == 0) printf("# QUDA: Not using preconditioning!\n");
   }
+  inv_param.verbosity= get_verbosity_type(Quda_invert_input.inv_verbosity);
```

Comment:
This should only be done on the coarser levels, the outer solver verbosity should be given by ```g_debug_level``` and is set above.